test/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const asyncExpectThrowError = async (prom: any, expectedError?: string) => {
+  let error;
+  try {
+    await prom();
+  } catch (_error) {
+    error = _error;
+  } finally {
+    expect(error).toBeTruthy();
+    if (expectedError) {
+      expect(error).toBe(expectedError);
+    }
+  };
+}
```

Comment:
We should use jest assertion here. 
Ref: https://jestjs.io/docs/en/expect#rejects